media-gfx/qimgv/metadata.xml

```diff
@@ -12,5 +12,6 @@
 
 	<use>
 		<flag name="video">Add support for gif/webm playback via libmpv</flag>
+		<flag name="blur">Add support for background blur</flag>
```

Comment:
thanks, please keep it sorted though.

Reply:
Hmmm, it seems upstream changed `KDE_BLUR` to `KDE_SUPPORT` https://github.com/easymodo/qimgv/commit/befe11787ae58ae079932dfea5ce9b5e54b93dd6

In this case, would it be better to have it as `kde` use flag rather than `blur`? In the case where future development adds more kde-specific functionality.

Reply:
Could you inquire this upstream? I don't think it would be limited to a Plasma session right now. If they plan for more in the future, yes, you could leave it at 'kde' at least for now. Mid-term I need to come up with a scheme to move away from that as a feature description for Plasma.

Reply:
I talked with upstream and they said they plan on adding more kde specifics in the future. So I will leave it as `kde`.